controllers/travel.controller.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+const Constants = {
+    Success: require("../constants/success.constant"),
+    Error: require("../constants/error.constant")
+};
+
+function okay(req, res) {
```

Comment:
nit: okay sounds a bit odd as a function name

Reply:
Whoops, this function was just for debugging - I'll remove it